internal/definitions/handler_tokenpool.go

```diff
@@ -92,6 +92,11 @@ func (dh *definitionHandler) handleTokenPoolDefinition(ctx context.Context, stat
 			}
 		}
 
+		if existing.ID.Equals(pool.ID) {
```

Comment:
I'm trying to understand how we got to this point with a duplicate ID. Your description indicates we are processing a `TokenPoolCreated` event, so we're coming in via `eventManager.TokenPoolCreated()` and then `definitionSender.DefineTokenPool()` and then here.

But in `eventManager.TokenPoolCreated()`, there is a check for an [existing pool](https://github.com/hyperledger/firefly/blob/62ee71f675ac5717b7c16e034b75ec1c69920d88/internal/events/token_pool_created.go#L162). Is that check failing to find the existing pool and allowing it to get down to here?

Reply:
Also worth confirming... are you running the latest release (v1.3.2)?

Reply:
Yes, running the latest v1.3.2 release